docs/HowTo/Configure-Slashing-Protection.md

```diff
@@ -144,6 +144,17 @@ You can include additional optional pruning configuration options. For example,
 
     Do not use slashing protection database pruning while [importing or exporting the database](#import-or-export-a-slashing-protection-database).
 
+## Health checks
```

Comment:
@wcgcyx please correct me if I'm wrong but I _think_ the health check exists even when slashing protection is enabled, so might make sense to put this section somewhere more general. Potentially a new page under "Monitor nodes" alongside https://docs.web3signer.consensys.net/en/latest/HowTo/Monitor/Metrics/?

Reply:
It's also related to the upcheck: https://docs.web3signer.consensys.net/en/latest/HowTo/Get-Started/Start-Web3Signer/#confirm-web3signer-is-running

Reply:
Ok, I renamed the section to "Slashing protection health check". I'll also add a note in the section regarding the upcheck.